docs/content/en/3.api/1.options.md

```diff
@@ -154,3 +154,19 @@ export default {
   }
 }
 ```
+
+## `dir`
+
+Default: `static`
+
+This option allows you to specify the location of the source images if using the `static` provider (IPX).
+
+For example you might want the source images in `assets/images` directory rather than the default `static` directory so the source images don't get copied into `dist` and deployed:
+
+```ts [nuxt.config.js]
+export default {
+  dir: 'assets/images'
+}
+```
+
+Note that changing the `dir` from the default should be done with caution. When using the static provider every image request/variant used during SSR rendering, crawling and generating can be collected and images generated. There are situations where that can't happen: uncrawled pages, modals and tabs, or client-only components. Usually the static provider falls back to the original source url but if it is not available users will get a 404 error.
```

Comment:
```suggestion
**Notes:**
- For `static` provider, if images weren't crawled during generation (unreachable modals, pages or dynamic runtime size), changing `dir` from `static` causes 404 errors.
- For `ipx` provider, make sure to deploy customized `dir` as well.
- For some providers (like vercel), using a directory other than `static/` for assets is not supported since resizing happens at runtime (instead of build/generate time) and source fetched from the `static/` directory (deployment URL)
```